core/src/main/java/pl/agh/edu/actor/slider/SliderComponent.java

```diff
@@ -0,0 +1,69 @@
+package pl.agh.edu.actor.slider;
+
+import java.util.function.Function;
+
+import com.badlogic.gdx.graphics.g2d.NinePatch;
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.ui.*;
+import com.badlogic.gdx.scenes.scene2d.utils.ChangeListener;
+import com.badlogic.gdx.utils.Align;
+
+import pl.agh.edu.actor.HotelSkin;
+
+public class SliderComponent extends Table {
+	private int SLIDER_WIDTH = 600;
+	private int HORIZONTAL_PADDING = 20;
+	private Label valueLabel;
+	private Slider slider;
+	private Skin skin;
+	private String suffix;
+	private Function<Float, Float> mapping;
```

Comment:
Po co nam funkcja przyjmująca za argument float i zwracająca float ?

Reply:
W zasadzie to niepotrzebna.